### DISCUSSION TITLE
Consultation 6. 5. 2026

### DISCUSSION THREAD

**Discussion by klimesm** [ctu-geoforall-lab-projects/dp-klimes-2026#26] [General]
- VENuS dataset a dlaždice s01 s rozlišením 5 m a zbylé 10 m
   - jde o level 1 produkt?
   - https://www.eoportal.org/satellite-missions/venus#ground-segment
    - > The ground processing center provides 3 product levels:  Level 1 will provide geolocated top of atmosphere reflectances with a sub-pixel multi-date registration. Level 2 provides surface reflectances ("Top of Canopy"). Its aim is to suppress as much as possible the atmospheric contribution to the signal. During this process, a high quality cloud and cloud shadow mask is generated. The Level 3 product is a 10 day composite of level 2 products. The aim is to provide the users with cloud-free, easy to use images of surface reflectances, every 10 days. The ground sampling distance of level 2 and 3 products will be 10 m, whereas the level 1 product is delivered at 5 m resolution.
    - tím pádem většina dlaždic je level 2 produkt (BOA), zatímco csen12 a většina datasetů co jsem našel je TOA
    - původně jsem myslel, že VENuS TOA má hodnoty do cca 6000, BOA do cca 800, ale jednalo se jen o outlier pixely. 
    - rozsahy jsou podobné - všechna venus data mají hodnoty pod 1000
    - tyto statistiky používám k experimentům, popíši, že vychází z kombinace L1 a L2 produků
- kam seznamy obrázků a tabulek - začátek? někdy vidím na konci, ale přirozeně bych dal na začátek
- learning rate
- MAJA masky  
    - je vše krom 0 mrak?
    - nemáte náhodou někde uložené venus masky z Figure 2 z článku? https://www.mdpi.com/2072-4292/14/20/5210
- experimenty - zhodnocení, detail popisu
    - dělat E1? dodá další krok v míře adaptace, ale je to práce navíc
    - pro transductive norm venus fixed je base model norm s2 fixed
    - ukázat pro každý experiment stejné testovací dlaždice nebo ty, které jsou zajímavé pro daný experiment
    - Exp ve figures porovnávám s ground truth - baseline do figure nedávám, aby zůstaly ukázky velké - je ok nebo přidat?
    - zbylé výsledky do příloh? do textu? nereportovat? hlavně vývoj val loss
- reportované metriky, seřazení výsledků 
- kam popsat, co jednotlivé exp popisují a jaké části freeze - v experimentech či v metodologii?

**Comment by klimesm** [comment]
vše je level 2 - 5m rozlišení je změnou orbity
seznamy na zacatek
lr někam popsat že bylo zkoušeno
maja mám teď lokálně jako png
E1 udělat
pro každý exp stejné dlaždice ideál - zahrnout vodu
zbylé výsledky do příloh
popis experimentů do methodology
popisy zkratek u obrázků a tabulek
dice val loss popis
upravit venus data popis
